packages/gatsby-plugin-lodash/src/gatsby-node.js

```diff
@@ -0,0 +1,18 @@
+const webpackLodashPlugin = require(`lodash-webpack-plugin`)
+
+// Add Lodash webpack plugin
+exports.modifyWebpackConfig = ({ config, stage }) => {
+  if (stage === `build-javascript`) {
+    config.plugin(`Lodash`, webpackLodashPlugin, null)
```

Comment:
`lodash-webpack-plugin` is pretty aggressive by default (disables a lot of feature sets). Is the plugin still configurable?

Reply:
oh not atm. Is there a non-aggressive default it could start with :-) And then let the plugin accept options?

Reply:
Hmmm or perhaps just remove the webpack plugin by default?

Reply:
I'm for encouraging its use, if it's configurable that should cover your bases.
When configurable you could provide your own defaults to mixin with others.

Reply:
What's a safe default so adding this plugin won't break any sites?

Reading some of the issues it sounds like it's a fairly manual tool to use? I.e. you need to know you're not using Lodash features and then remove them?

Reply:
E.g. don't want this to happen https://github.com/lodash/lodash-webpack-plugin/issues/114#issuecomment-315768773

Reply:
You could flip all features on then have folks opt-out (inverse).

Reply:
Ok, opt-out sounds like the right approach. Thanks for stepping in and helping!